### DISCUSSION TITLE
recenter cursor position

### DISCUSSION THREAD

**Discussion by FelicityF** [jspsych/jsPsych#1013] [Q&A] [answered]
I'm wondering whether it is possible to recenter the cursor after each trial? Thus, participant sees the fixation cross > four pictures are presented at the screen while listening to a spoken sentence and participant has to click on one of the four pictures. When the pictures are presented, I want to cursor to be in the same position as the fixation cross. 

Thank you!

**Comment by becky-gilbert** [comment, marked as answer]
Hi @FelicityF! I don't think this is possible, see: https://stackoverflow.com/questions/4752501/move-the-mouse-pointer-to-a-specific-position One work-around might be to have a button in the center of the screen that participants have to click on to start the trial, which means that their cursor will be roughly in that position when the trial starts.

**Reply by FelicityF** [reply]
@becky-gilbert Thank you for the great suggestion! I have one question though: I read that it is possible to resize the button, but I'm not entirely sure how to do that. Do you have any suggestions? This is my code for the fixation cross: 

```
var fixation = {
type: 'html-button-response',
stimulus: '<div style="font-size:12px"></div>',
choices: '+',
response_ends_trial: true
}
```

**Reply by becky-gilbert** [reply]
You can change the button style using the `button_html` parameter (see [html-button-response plugin docs](https://www.jspsych.org/plugins/jspsych-html-button-response/#parameters)). The default button HTML is `<button class="jspsych-btn">%choice%</button>`. The `%choice%` string will be replaced with the value(s) from your `choices` parameter (i.e. the "+" in your case). 
To make the button bigger, you could specify your own button HTML with some in-line style, e.g.
```javascript
button_html: '<button class"jspsych-btn" style="width:300px; font-size:50px;">%choice%</button>'
``` 
Also, the value of `choices` has to be an array, even if it only contains one element. So all together your fixation might look like this:
```javascript
var fixation = {
  type: 'html-button-response',
  stimulus: '<div style="font-size:12px"></div>',
  choices: ['+'],
  response_ends_trial: true,
  button_html: '<button class"jspsych-btn" style="width:300px; font-size:50px;">%choice%</button>'
}
```

**Reply by FelicityF** [reply]
@becky-gilbert Thank you so much! This works :-)
